submissions/mustbefail/dom-api-task/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+      integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
+      crossorigin="anonymous"
+    />
+    <script src="./script.js" defer></script>
+    <link rel="stylesheet" href="index.css" />
+    <title>Movie by genre</title>
+  </head>
+  <body>
+    <header>
+      <h1>Movies by genre</h1>
+    </header>
+    <div class="sidebar">
+      <div class="menu-chevron">
+        <a href="#"><i class="fas fa-chevron-right"></i></a>
+      </div>
+      <nav>
+        <ul>
+          <h3 class="sidebar-header">Movie Genres</h3>
```

Comment:
You can move this `h3` tag outside the `ul` tag because it can accept only `li` tags as children.